src/main/java/com/nike/cerberus/domain/environment/StackName.java

```diff
@@ -24,8 +24,16 @@
     VAULT("vault"),
     CMS("cms"),
     GATEWAY("gateway"),
+    VPC("vpc"),
+    DATABASE("database"),
+    SECURITY_GROUPS("security-groups"),
+    LOAD_BALANCER("load-balancer"),
+    ROUTE53("route53"),
+    WAF("web-app-firewall"),
     @Deprecated
-    RDSBACKUP("rdsbackup"); // TODO: need to remove but casually deleting will cause JSON parse error
+    RDSBACKUP("rdsbackup"), // TODO: need to remove but casually deleting will cause JSON parse error
+    @Deprecated
+    DEPRECATED_BASE("base");  // TODO: remove this class when old backup and restore is removed
```

Comment:
I don't see why we can't delete this stuff now and use the non-highlander version of the CLI for restoring old backups.